users/templates/mail/welcome_and_next_steps.html

```diff
@@ -26,9 +26,7 @@ <h3>Your services</h3>
     <li>{{ servicesubscription.service.name }} -
         {{ servicesubscription.service.cost_string }} /
         {{ servicesubscription.service.period_string }}
-        {% if servicesubscription.service.id == settings.DEFAULT_ACCOUNT_SERVICE %}
-          <strong>{% trans 'Bank reference number' %}: {{ user.reference_number }}</strong>
-        {% endif %}</li>
+        <strong>{% trans 'Bank reference number' %}: {{  servicesubscription.reference_number }}</strong>
```

Comment:
This will now show the reference number for all services. Was this intended? I was under the impression that the reference number was not wanted for the yearly fee?

Reply:
Yes, it's intended. Less special cases.